src/main/java/io/github/thebusybiscuit/slimefun4/api/events/AncientAltarOutputEvent.java

```diff
@@ -0,0 +1,70 @@
+package io.github.thebusybiscuit.slimefun4.api.events;
+
+
+import io.github.thebusybiscuit.slimefun4.implementation.items.altar.AncientAltar;
+import org.bukkit.block.Block;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+
+/**
+ * This {@link Event} is fired before an item is dropped by an {@link AncientAltar}.
+ * Cancelling this event will make the {@link AncientAltar} drop no item after the recipe is finished.
+ *
+ * @author Tweep
+ *
+ */
+
+public class AncientAltarOutputEvent extends Event implements Cancellable {
```

Comment:
This should probably be a player event. I'd like to get the player here.

Reply:
Sure thing, hang on a second